ui/raidboss/data/07-dt/dungeon/ihuykatumu.ts

```diff
@@ -0,0 +1,193 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'Ihuykatumu',
+  zoneId: ZoneId.Ihuykatumu,
+  timelineFile: 'ihuykatumu.txt',
+  triggers: [
+    // ** Prime Punutiy ** //
+    {
+      id: 'Ihuykatumu Prime Punutiy Punutiy Press',
+      type: 'StartsUsing',
+      netRegex: { id: '8E8C', source: 'Prime Punutiy', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Decay',
+      type: 'StartsUsing',
+      netRegex: { id: '8E99', source: 'Ihuykatumu Flytrap', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get under Flytrap',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Inhale',
+      type: 'StartsUsing',
+      netRegex: { id: '8E8E', source: 'Prime Punutiy', capture: false },
+      durationSeconds: 10, // prolonged ground damage
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Stay out of inhale',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Shore Shaker',
+      type: 'StartsUsing',
+      netRegex: { id: '8EA2', source: 'Prime Punutiy', capture: false },
+      durationSeconds: 6,
+      response: Responses.getOutThenIn(),
+    },
+
+    // ** Drowsie ** //
+    {
+      id: 'Ihuykatumu Drowsie Uppercut',
+      type: 'StartsUsing',
+      netRegex: { id: '98DC', source: 'Drowsie' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Ihuykatumu Drowsie Wallop Small',
+      type: 'StartsUsing',
+      netRegex: { id: '8E7F', source: 'Ihuykatumu Ivy', capture: false },
+      delaySeconds: 2,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge Ivy cleaves (small)',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Drowsie Wallop Large',
+      type: 'StartsUsing',
+      netRegex: { id: '8E82', source: 'Ihuykatumu Ivy', capture: false },
+      delaySeconds: 2,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge Ivy cleaves (big)',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Drowsie Sneeze',
+      type: 'StartsUsing',
+      netRegex: { id: '8E7B', source: 'Drowsie', capture: false },
+      response: Responses.getBehind(),
```

Comment:
I suppose it's not terribly unreasonable to have a trigger for this, but I wonder whether `awayFromFront` might fit better, even as wide as it is.

Reply:
> I suppose it's not terribly unreasonable to have a trigger for this, but I wonder whether `awayFromFront` might fit better, even as wide as it is.

I was pretty indifferent about this one tbh (probably for the same reasons).  I'm fine with `awayFromFront` (or yoinking it if others prefer).